packages/ui/src/models/visualization/base-visual-entity.ts

```diff
@@ -188,6 +188,10 @@ export interface NodeInteraction {
   canRemoveStep: boolean;
   canRemoveFlow: boolean;
   canBeDisabled: boolean;
+  canBeCopied: boolean;
+  canBePastedAsNextStep: boolean;
+  canBePastedAsChild: boolean;
+  canBePastedAsSpecialChild: boolean;
```

Comment:
Idea: What do you think of removing this interaction set, as the copy-paste interactions don't depend on the node itself, but on the target as well.

Perhaps we could move this logic to the ContextMenu part, so we don't need to add these properties here.